manifests/concourse-manifest/spec/manifest_generation_spec.rb

```diff
@@ -34,18 +33,10 @@ def merge_fixtures(fixtures)
     ).to eq("https://" + fixtures["terraform_outputs"]["concourse_dns_name"])
   end
 
-  it "gets values from generated secrets" do
-    # fixture_atc_password = fixtures["jobs"][0]["templates"][2]["properties"]["basic_auth_password"]
-
-    expect(
-      manifest_with_defaults["jobs"][0]["templates"][1]["properties"]["basic_auth_password"]
-    ).to eq(fixtures["secrets"]["concourse_atc_password"])
-  end
-
-  it "gets values from predefined secrets" do
+  it "gets values from secrets" do
     expect(
       atc_template.fetch("properties").fetch("basic_auth_password")
```

Comment:
They are. If you look at https://github.com/alphagov/paas-bootstrap/blob/master/manifests/concourse-manifest/concourse-base.yml#L78 you can see that `jobs[0]` Is the concourse job, and `templates[1]` is the atc template on that job. Which is what the `atc_template` rspec helper pulls out at the top of this file (although it pulls them out by name, which is much more robust.)